### PR TITLE
Updated read.me and contributing.md

### PR DESCRIPTION
**Pull request by vcoisne**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-blog-post.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ Issue Title: [PROPOSAL] Blog post title
 * Categories:
 * Deadlines: are there any deadlines for this? do we need to wait for any other event?
 
+## Blog process
+
+- Create a proposal on this blog repo
```
Comment (dpordomingo) [inline .github/ISSUE_TEMPLATE/new-blog-post.md]:
```suggestion
- Create a proposal on this blog repo [as a new issue](https://github.com/src-d/blog/issues)
```

File: .github/ISSUE_TEMPLATE/new-blog-post.md [hunk 2]
```diff
@@ -12,6 +12,15 @@ Issue Title: [PROPOSAL] Blog post title
...
+- Get proposal accepted by DevRel
+- Create a copy of [this gdoc](https://docs.google.com/document/d/1BxhHlkxytm-xCDVy4vRkdyjqwe9w7pIWoauaLpMMreE/edit#heading=h.rujpt9lvqbj8) & update the template header 
+- Add the draft of you blog post directly in gdoc with grammarly enabled
```
Comment (dpordomingo) [inline .github/ISSUE_TEMPLATE/new-blog-post.md]:
```suggestion
- Add the draft of you blog post directly in gdoc with grammarly enabled
- Share the draft company wide, and paste its link in the issue containing your proposal in order to let the people to participate in the review process.
```

Comment (dpordomingo) [inline .github/ISSUE_TEMPLATE/new-blog-post.md]: Should the draft be stored in any special directory?

File: .github/ISSUE_TEMPLATE/new-blog-post.md [hunk 3]
```diff
@@ -12,6 +12,15 @@ Issue Title: [PROPOSAL] Blog post title
 * Categories:
 * Deadlines: are there any deadlines for this? do we need to wait for any other event?
 
+## Blog process
+
+- Create a proposal on this blog repo
+- Get proposal accepted by DevRel
+- Create a copy of [this gdoc](https://docs.google.com/document/d/1BxhHlkxytm-xCDVy4vRkdyjqwe9w7pIWoauaLpMMreE/edit#heading=h.rujpt9lvqbj8) & update the template header
```
[the remark this answers is not in the document]
Comment (vcoisne) [reply .github/ISSUE_TEMPLATE/new-blog-post.md]: yes updated, thanks

File: .github/ISSUE_TEMPLATE/new-blog-post.md (hunk 2 above)
Comment (vcoisne) [reply .github/ISSUE_TEMPLATE/new-blog-post.md]: Yes in a google doc folder called Blog posts under DevRel

Comment (vcoisne) [reply .github/ISSUE_TEMPLATE/new-blog-post.md]: LGTM

File: .github/ISSUE_TEMPLATE/new-blog-post.md (hunk 1 above)
Comment (vcoisne) [reply .github/ISSUE_TEMPLATE/new-blog-post.md]: LGTM

**Comment by vcoisne** [comment]
@dpordomingo I accepted your suggestions, I meant the PR but maybe that should be @campoy

File: .github/ISSUE_TEMPLATE/new-blog-post.md [hunk 4]
```diff
@@ -12,6 +12,16 @@ Issue Title: [PROPOSAL] Blog post title
 * Categories:
 * Deadlines: are there any deadlines for this? do we need to wait for any other event?
 
+## Blog process
+
+- Create a proposal on this blog repo [as a new issue](https://github.com/src-d/blog/issues)
```
Comment (dpordomingo) [inline .github/ISSUE_TEMPLATE/new-blog-post.md]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/new-blog-post.md (hunk 2 above)
Comment (dpordomingo) [reply .github/ISSUE_TEMPLATE/new-blog-post.md]:
```suggestion
- Add the draft of you blog post directly in gdoc with grammarly enabled, and store it under [DevRel>BlogPosts](https://drive.google.com/drive/folders/1PGIqQ284kAXWgY50Nco9lRF6VIYGyxKg)
```

File: .github/ISSUE_TEMPLATE/new-blog-post.md [hunk 5]
```diff
@@ -12,6 +12,16 @@ Issue Title: [PROPOSAL] Blog post title
...
+- Get proposal accepted by DevRel
+- Create a copy of [this gdoc](https://docs.google.com/document/d/1BxhHlkxytm-xCDVy4vRkdyjqwe9w7pIWoauaLpMMreE/edit#heading=h.rujpt9lvqbj8) & update the template header 
+- Add the draft of you blog post directly in gdoc with grammarly enabled
+- Share the draft company wide, and paste its link in the issue containing your proposal in order to let the people to participate in the review process.
```
Comment (dpordomingo) [inline .github/ISSUE_TEMPLATE/new-blog-post.md]:
```suggestion
- Once your draft is ready for being reviewed, share it company-wide, and paste its link in this issue in order to let the people to participate in the review process.
```

**Pull request merged**